src/qnngds/design.py

```diff
@@ -700,33 +700,31 @@ def ntron_cell(
             fill_pad_layer=self.fill_pad_layer,
         )
 
-    def snspd_cell(
+    def snspds_cell(
```

Comment:
Note: There is no picture for this function on the website

Reply:
see issue #37